ArchiSteamFarm/Core/ASF.cs

```diff
@@ -285,6 +285,31 @@ public static class ASF {
 				return null;
 			}
 
+			ArchiLogger.LogGenericInfo(Strings.VerifyingChecksumWithRemoteServer);
```

Comment:
Should this logic be triggered in case of the custom build?

Reply:
Custom builds already have `CanUpdate => false`, so they never update to anything to begin with. I don't believe it makes sense checking this again.